scapy/utils.py

```diff
@@ -543,26 +598,29 @@ def strand(s1, s2):
 try:
     socket.inet_aton("255.255.255.255")
 except socket.error:
-    def inet_aton(x):
-        if x == "255.255.255.255":
+    def inet_aton(ip_string):
+        # type: (str) -> bytes
+        if ip_string == "255.255.255.255":
             return b"\xff" * 4
         else:
-            return socket.inet_aton(x)
+            return socket.inet_aton(ip_string)
 else:
     inet_aton = socket.inet_aton
 
 inet_ntoa = socket.inet_ntoa
 
 
 def atol(x):
+    # type: (str) -> int
     try:
         ip = inet_aton(x)
     except socket.error:
         ip = inet_aton(socket.gethostbyname(x))
-    return struct.unpack("!I", ip)[0]
+    return struct.unpack("!I", ip)[0]  # type: ignore
```

Comment:
Why is `type: ignore` needed here?

Reply:
`unpack` is defined super loosely https://github.com/python/typeshed/blob/5d553c9584eb86793cfa61019ddee5c7b62bc286/stdlib/2and3/struct.pyi#L23
It's type is just `Any`? We could cast it though